app/components/Send/SendPanel/SendSuccess/index.jsx

```diff
@@ -14,43 +14,49 @@ type Props = {
   txid: string
 }
 
-const SendSuccess = ({ sendRowDetails, txid }: Props) => {
-  const numberOfItems = sendRowDetails.length
+export default class SendSuccess extends React.Component<Props> {
+  txTimestamp: string
 
-  return (
-    <section>
-      <div className={styles.sendSuccessHeader}>
-        <CheckMarkIcon className={styles.sendSuccessHeaderIcon} />
-        <div className={styles.sendSuccessHeaderInfo}>
-          <h1 className={styles.sendSuccessHeaderInfoText}>
-            {numberOfItems} {pluralize('Transfer', numberOfItems)} completed
-          </h1>
-          <p className={styles.sendSuccessParagraphText}>
-            {createFormattedDate()}
-          </p>
-          <p className={styles.sendSuccessParagraphText}>
-            Transaction ID: {txid}
-          </p>
+  componentDidMount() {
+    this.txTimestamp = createFormattedDate()
```

Comment:
minor - perhaps `txFormattedDate` would be a more appropriate name for this variable.